src/eradiate_plugins/tests/bsdfs/test_mqdiffuse.py

```diff
@@ -124,7 +124,7 @@ def test_eval(variant_scalar_rgb, plugin, theta_o, phi_o, theta_i, phi_i, expect
     assert dr.allclose(value, expected * dr.cos(theta_o), atol=1e-6)
 
 
-def test_chi2(variants_vec_backends_once_rgb, tmp_path):
+def test_chi2(variant_llvm_rgb, tmp_path):
```

Comment:
I may have overlooked this when changing the variants that are actually not compiled with the default Eradiate cmake preset. I might revert this

Reply:
No I did not. The variants_vec_backends_once_rgb results in skipped tests using the default Eradiate preset. I thus specialized them so that the tests actually execute

Reply:
Maybe there's something else I misunderstood with these fixtures?

Reply:
Good point: it's a good idea to get back to the [definition of this fixture](https://github.com/eradiate/mitsuba3/blob/68b97fc2453558c248faa7d4cbb5befc06dbc30e/src/conftest.py#L119), and to the [definition of default Eradiate variants](https://github.com/eradiate/mitsuba3/blob/68b97fc2453558c248faa7d4cbb5befc06dbc30e/CMakePresets.json#L22). 

First of all: My memory wasn't right, because, for a reason I can't remember, we compile both `llvm_ad_rgb` (Mitsuba default) and `llvm_rgb`. This is probably a mistake because neither kernel testing nor any Eradiate application requires `llvm_rgb`. So we could stick with `llvm_ad_rgb`.

Second point I got wrong: the `vec_backends_once_rgb` picks one RGB variant per vector backend, so in our case, that would be `llvm_rgb` *or* `llvm_ad_rgb` (we don't compile a CUDA variant). I though it would pick one at all (*i.e.* choose arbitrarily between CUDA and LLVM, which is actually pretty silly).

So, coming to the expected behaviour: the idea is that the χ² test should be executed once with one of the compiled LLVM variants, and skipped for the CUDA part. But if we ever start compiling a CUDA variant, the test will be executed.

Reply:
Now I realize where we might have a different opinion: in my view, it's not a problem to have a skipped test here, maybe this is what bothered you in the first place?

Reply:
No I simply did not realise that only a CUDA test was skipped, I thought the entire $\chi²$ test was ignored. I will revert these changes and stick llvm_**ad** variants.